apps/wallet/src/ui/app/staking/delegation-detail/DelegationDetailCard.tsx

```diff
@@ -75,6 +75,10 @@ export function DelegationDetailCard({
         staked: stakedId,
     }).toString()}`;
 
+    const inActiveValidator = !system?.activeValidators?.find(
```

Comment:
Is this if the wallet is in the active list, or if it's inactive? Can we make this more obvious?

Reply:
rename to `isValidatorInactive`